tests/unit/backend/corpora/common/entities/test_dataset.py

```diff
@@ -93,16 +94,51 @@ def test__update__ok(self):
             s3_uri="some_uri",
         )
         deployment_directory_params = dict(url="test_url")
+        processing_status = dict(
+            upload_status=UploadStatus.UPLOADING,
+            upload_progress=1 / 9,
+            validation_status=ValidationStatus.NA,
+            conversion_loom_status=ConversionStatus.NA,
+            conversion_rds_status=ConversionStatus.NA,
+            conversion_cxg_status=ConversionStatus.NA,
+            conversion_anndata_status=ConversionStatus.NA,
+        )
         dataset_params = BogusDatasetParams.get()
         dataset = Dataset.create(
-            **dataset_params, artifacts=[artifact_params] * 1, deployment_directories=[deployment_directory_params] * 1
+            **dataset_params,
+            artifacts=[artifact_params] * 1,
```

Comment:
What does the `* 1` do here?

Reply:
Hmm nothing I don't think. Looks like it's maybe copied from a pattern where different counts of entities are added.